src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -184,9 +210,6 @@ public ResponseEntity<Set<FaqDTO>> getFaqForCourse(@PathVariable Long courseId)
     @EnforceAtLeastStudent
```

Comment:
Why dont we have the course check here? 

```suggestion
    @EnforceAtLeastStudentInCourse
```